src/Docker.PowerShell/Cmdlets/DockerFactory.cs

```diff
@@ -14,10 +14,12 @@ public class DockerFactory
     public static DockerClient CreateClient(string HostAddress, string CertificateLocation)
     {
         HostAddress = HostAddress ?? Environment.GetEnvironmentVariable("DOCKER_HOST") ?? "npipe://./pipe/docker_engine";
-        CertificateLocation = CertificateLocation ?? Environment.GetEnvironmentVariable("DOCKER_CERT_PATH");
+        
+        bool shouldUseTLS = !string.IsNullOrEmpty(CertificateLocation) || "1" == Environment.GetEnvironmentVariable("DOCKER_TLS_VERIFY");
+        CertificateLocation = CertificateLocation ?? Environment.GetEnvironmentVariable("DOCKER_CERT_PATH");        
 
         Credentials cred = null;
-        if (!string.IsNullOrEmpty(CertificateLocation))
+        if (shouldUseTLS)
         {
             //BUGBUG(swernli) - Remove this later in favor of something better.
             ServicePointManager.ServerCertificateValidationCallback += (o, c, ch, er) => true;
```

Comment:
I think we need to fix this to actually do the verification before we merge this